StreamChatClient_v3/Workers/EventObservers/MemberEventObserver_Tests.swift

```diff
@@ -0,0 +1,126 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatClient_v3
+import XCTest
+
+final class MemberEventObserver_Tests: XCTestCase {
```

Comment:
Do you think we need such detailed tests for this? It duplicates a lot the tests of `EventObserver`. Maybe it would be enough to just test the `filter` is set up correctly?

Reply:
fixed in eaa477096e31097b6278c929970beb799580a3ae

Reply:
We either didn't understand each other or this is not fixed :)

Reply:
Hm, you are right 🤔  Did these changes but lost somewhere on the go. Finally fixed in b40eab6d84232560c2058e5f5c22e7f0d89ed128